robusta_krr/core/integrations/prometheus/metrics_service/prometheus_metrics_service.py

```diff
@@ -79,6 +79,10 @@ def __init__(
         logger.info(f"Using {self.name} at {self.url} for cluster {cluster or 'default'}")
 
         headers = settings.prometheus_other_headers
+        # FIXME: not the right place, Mimir specific
+        headers.update(
+            {"X-Scope-OrgID": "anonymous"}
```

Comment:
That's probably the wrong place, but I don't know where to put this.  It is also strange that it is needed: passing in a Mimir URL using `--prometheus-url` succeeds also without this header.  Also, this should be configurable: Mimir has multi-tenant capabilities and this header identifies the tenant (AFAIK).

Reply:
L81 allows you to set other headers via command line arguments